syncwatcher.go

```diff
@@ -459,6 +450,23 @@ func relativePath(path string, folderPath string) string {
 	return path
 }
 
+// Returns a function to test whether a file should be ignored
+// Ignorefile must contain the absolute path to ".stignore". The returned
+// function takes the path of the file to be tested relative to its folders root
+func createIgnoreTest(folderPath string) func(relPath string) bool {
+	ignores := ignore.New(false)
+	ignores.Load(filepath.Join(folderPath, ".stignore"))
+	return func(relPath string) bool {
+		if strings.HasPrefix(relPath, versionsFolder) { return true }
```

Comment:
You're right of course. Ignoring only `.stversions` is not enough. The best option I could find for extracting the leading directory from `relPath` is

`strings.SplitN(relpath, pathSeparator, 2)[0]`